content/pages/health-care/about-va-health-care.md

```diff
@@ -225,7 +225,7 @@ If you have other forms of health care coverage (like a private insurance plan,
 <div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
```

Comment:
Should there be a +1 before 877-222-VETS?

Reply:
Yup updated.